packages/commons-ui-core/src/ImageButton/ImageButton.js

```diff
@@ -38,10 +38,10 @@ const ImageRoot = styled("span", {
 }));
 
 const ImageButton = React.forwardRef(function ImageButton(props, ref) {
-  const { src } = props;
-  let image;
+  const { children, src, onClick } = props;
```

Comment:
```suggestion
  const { children, src, ...other } = props;
```

`onClick` should go to the button (`ImageButtonRoot`) as part of `other`.